shared/macros-ansible.jinja

```diff
@@ -418,7 +418,7 @@ The macro requires following parameters:
 - syscalls: list of syscalls supplied as a list ["syscall1", "syscall2"] etc.
 - key: a key to use as rule identifier.
 #}}
-{{% macro ansible_audit_auditctl_add_syscall_rule(arch="", syscalls=[], key="") -%}}
+{{%- macro ansible_audit_auditctl_add_syscall_rule(arch="", syscalls=[], key="") -%}}
```

Comment:
Any reason for the auditctl macro to have the leading dash `-` and augenrules macro to miss it?

Reply:
Thanks, I removed it, I tried to fix something but this approach didn't help and I forgot about it.